tests/test_wave_01.py

```diff
@@ -116,14 +116,15 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     updated_data = watch_movie(janes_data, MOVIE_TITLE_1)
 
     # Assert
-    assert len(updated_data["watchlist"]) is 0
-    assert len(updated_data["watched"]) is 1
+    assert len(updated_data["watchlist"]) == 0
+    assert len(updated_data["watched"]) == 1
     
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
-
-@pytest.mark.skip()
+    assert MOVIE_TITLE_1 in updated_data["watched"][0].values() #my_test
```

Comment:
This assert works, but consider making it a bit more specific and explicit. Something like: 

```
assert MOVIE_TITLE_1 == updated_data["watched"]["title"]
```